client/templates/joining.html

```diff
@@ -1,13 +1,13 @@
-<div id="signup-container" class="namestuff">
-    <h2>Hi {{enteredName}}!</h2>
+<div id="room-join-container" class="namestuff">
+    <h2 id="user-name-text">Hi {{enteredName}}!</h2>
```

Comment:
If you are using protractor, you can reference this element using it's angular binding. No need to give it an Id which would otherwise have no purpose.

`element(by.binding('enteredName'))`